charts/barman/README.md

```diff
@@ -73,7 +73,7 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 
 Adfinis fights for a software world that is more open, where the quality is
 better and where software must be accessible to everyone. This chart
-is part of the action behind this comitment. Feel free to
+is part of the action behind this commitment. Feel free to
```

Comment:
I'd merge master into this

Reply:
ah yes, sure. I'll do it. This PR is still WIP :smile:

Reply:
or rebase onto master... I'm not quite sure why/how the issue that unrelated changes show up in a PR can be triggered since it never happened to me :(

Reply:
rebasing is the reviewer's nightmare.